person.rb

```diff
@@ -1,3 +1,5 @@
+require './corrector'
+
 class Person
   attr_accessor :name, :age
   attr_reader :id
```

Comment:
- [ ] kindly make sure the Person class inherits from the Namable class. also, It should have a method `correct_name`. :point_down: 

```ruby
class Person < Nameable
  .
  .

    def correct_name
       name
    end
  .
  .
end
```